styles.css

```diff
@@ -382,3 +383,409 @@ body {
   justify-content: space-between;
   width: 90%;
 }
+
+.your-email,
+.your-name,
+.your-message {
+  color: transparent;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Also this is how your `about` page looks when we have a `min-width: 768px`

![Annotation 2023-03-03 230](https://user-images.githubusercontent.com/57408419/222835788-fe4c1a6f-9dca-4833-81c9-e85921a8da36.jpg)

Kindly, make sure you fixed the styles to replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=ANytl328Q4Wuw0jZ-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.). 👍